test/jdk/java/text/BreakIterator/BreakIteratorTest.java

```diff
@@ -101,6 +101,15 @@ public BreakIteratorTest()
     // general test subroutines
     //=========================================================================
 
+    private int getErrorCount() {
+        return errorCount;
+    }
+
+    private void err(String message) {
+        errorCount++;
+        fail(message);
+    }
+
```

Comment:
I think the previous code was set up so that if forward and backward did not pass, than expected and actual was skipped. Since IntlTest could be ran with -nothrow which allowed for tests to continue even if an error was encountered. 

Since that is no longer the case, I simply adjusted `compareFragmentLists` to fail if an error is found.